integration-libs/epd-visualization/root/services/index.ts

```diff
@@ -0,0 +1,4 @@
+export * from './scene-node-to-product-lookup/index';
+// Don't include storage-api
+export * from './visualization-lookup/index';
+// Don't include visualization-api
```

Comment:
Why don't we want to expose in the public api the services responsible for the communication with the backend API?

Reply:
Fair enough. It would be good to mention that fact in the official documentation of your feature in https://github.com/SAP/spartacus-docs https://sap.github.io/spartacus-docs/

For more on how/where to document, please contact @gladius-mtl